test/travis.yml

```diff
@@ -65,6 +65,8 @@ jobs:
     - <<: *stage_quick
       env: CH_BUILDER=docker CH_TEST_SCOPE=quick
 
+    - <<: *stage_builders
+      env: CH_BUILDER=none
```

Comment:
I'd like to keep the stages to 5 jobs, because that's what we can run in parallel, so the 6th job makes it take twice as long. I wonder if we should omit `buildah-setuid`, because that seems perhaps the least interesting of the Buildah flavors?

Reply:
For our purposes I think testing without a builder is more important than testing the buildah setuid subset.